gulpfile.js

```diff
@@ -39,7 +40,10 @@ var REACT_OPTS = {es6: true},
 // Runs JSX, ES6 transforms, browserify, adds sourcemaps, and notifies the
 // livereload server that JS has changed.
 gulp.task('js', function() {
-  var bundler = watchify(browserify(BROWSERIFY_OPTS));
+  livereactload.monitor(PATHS.dest + 'bundled.js')
+
+  var bundler = watchify(browserify(_.extend({transform: [livereactload]}, BROWSERIFY_OPTS)));
+  rebundle();
```

Comment:
this looks a bit odd — you call `rebundle()` before it's defined, then immediately call it again?